debian/control

```diff
@@ -9,7 +9,8 @@ Build-Depends: check,
                libboost-dev (>= 1.41),
                libboost-program-options-dev (>= 1.41),
                libssl-dev,
-               scons (>= 2)
+               scons (>= 2),
+               libasio-dev
```

Comment:
Minimum version requirement would make a build fail on older debian/ubuntu targets where recent enough libasio-dev is not available. This looked like good enough compromise: Have a build dependency on libasio-dev library, but don't use it if it is not recent enough.